static/views/common-css.html

```diff
@@ -0,0 +1,45 @@
+{{ define "common-css" }}
```

Comment:
I think would be best to call this dark-mode-css
```suggestion
{{ define "dark-mode-css" }}
```

I am looking to add many more of these templates going forward and would want to avoid having massive "common" css into a single file in the future :)